toolkit/cli/node-commands/src/lib.rs

```diff
@@ -234,6 +396,32 @@ fn setup_log4rs() -> Result<(), Box<dyn std::error::Error + Send + Sync>> {
 	Ok(())
 }
 
+/// Utility function for printing command execution results.
```

Comment:
I have a question here.
To get to the conclusion about what this function does, is it easier to get through this comment or read the function itself?